### DISCUSSION TITLE
Better accommodate other time zones in meeting schedule

### DISCUSSION THREAD

**Discussion by makkes** [open-gitops/project#103] [General]
I quickly discussed this with @scottrigby on Tuesday but thought it might be worth bringing to the channel, too: I would love for this group to establish something like an alternating-cadence meeting schedule to make it easier for European folks to join in. At Flux, we have "early" and "late" meetings to accomodate for different time zones. Would you be up for doing something like that for this group as well?

([original Slack thread](https://cloud-native.slack.com/archives/C026D3RE7FB/p1652965122258889) preceeding this discussion)

**Comment by nishadmehendale** [comment]
+1

**Comment by niklasmtj** [comment]
This sounds interesting. How do you handle the information that results in the meetings? For example when in the early meeting something is discussed which is also discussed in the late meeting. Is there something written down in the mean time so the "late" meeting have all the information that was discussed earlier? 

Maybe it is a good idea to write down the results in a way that the other group have enough information about what was said. 

The summer time 9pm meetings are oookay but a little earlier wouldn't hurt. But then we get straight into managing time zones 😀 But I'm open to this idea 👍🏼

**Reply by makkes** [reply]
Flux maintains an [agenda doc](https://docs.google.com/document/d/1l_M0om0qUEN_NNiGgpqJ2tvsF2iioHkaARDeh6b70B0/edit?usp=sharing) and there's always an overlap between people attending the early and the late meeting so they can transport important items from one to the other.
